README.md

```diff
@@ -72,18 +82,18 @@ class CalculatorSteps {
   }
 
   @Then(/^The result is (\d+)$/)
-  public stepResult(expectedResult: number) {
-    expect(this.result).toStrictEqual(expectedResult);
+  public stepResult(expectedResult: string) {
```

Comment:
Shouldn't we give an example of `@Types` using this example?

Reply:
We absolutely could, but considering we are doing a "complete" documentation (powered by Docusaurus), I'd rather leave the code as-is, to show the default behavior without additional code like `@Types([Number])`, and add a reference to the specific doc.